__tests__/app.test.js

```diff
@@ -9,171 +9,171 @@ afterAll(() => {
-                })
-        });
-    });
-    describe('/api/users', () => {
-        test('status:200 returns a single array', () => {
-            return request(app)
-                .get('/api/users')
-                .expect(200)
-                .then(res => {
-                    expect(res._body).toBeInstanceOf(Array);
-                });
-        });
-        test('status:200 returns a array of objects containing property: username, name & avatar_url', () => {
-            return request(app)
-                .get('/api/users')
-                .expect(200)
-                .then(res => {
-			expect(res._body.length).toBe(4)
-                    res._body.forEach(news => {
-                        expect(news).toEqual(
-                            expect.objectContaining({
-                                username: expect.any(String),
-                                name: expect.any(String),
-                                avatar_url: expect.any(String)
-                            }),
-                        );
-                    });
-                });
-        });
-        test('status:404 Not Found', () => {
-            return request(app)
-                .get('/api/userss')
-                .expect(404)
-                .then(({ body }) => {
-                    expect(body.msg).toBe('Not found');
-                });
-	
-        });
-    });
+	describe('/api/topics', () => {
+		test('status:200 returns a single array', () => {
+			return request(app)
+				.get('/api/topics')
+				.expect(200)
+				.then(res => {
+					expect(res._body).toBeInstanceOf(Array);
+				});
+		});
+		test('status:200 returns a array of objects containing property: slug & description', () => {
+			return request(app)
+				.get('/api/topics')
+				.expect(200)
+				.then(res => {
+					res._body.forEach(news => {
+						expect(news).toEqual(
+							expect.objectContaining({
+								slug: expect.any(String),
+								description: expect.any(String),
+							}),
+						);
+					});
+				});
+		});
+		test('status:404 Not Found', () => {
+			return request(app)
+				.get('/api/topic')
+				.expect(404)
+				.then(({ body }) => {
+					expect(body.msg).toBe('Not found');
+				});
+		});
+	});
+	describe('/api/articles/:article_id', () => {
+		test('status:200 returns a single object', () => {
+			return request(app)
+				.get('/api/articles/1')
+				.expect(200)
+				.then(res => {
+					expect(res.body).toBeInstanceOf(Object);
+				});
+		});
+		test('status:200 returns a object with the required properties', () => {
+			return request(app)
+				.get('/api/articles/1')
+				.expect(200)
+				.then(({ body }) => {
+					expect(body).toEqual({
+						article_id: 1,
+						title: 'Living in the shadow of a great man',
+						topic: 'mitch',
+						author: 'butter_bridge',
+						body: 'I find this existence challenging',
+						created_at: '2020-07-09T20:11:00.000Z',
+						votes: 100,
+						comment_count: 11,
```

Comment:
in an ideal world we want a brand new test for comment count and not to modify previous tests. Previous tests should not fail if we add a new column to our database so think about how we could use another testing method in the future to prevent fails from happening.